gitclone/git.go

```diff
@@ -266,6 +266,28 @@ func manualMerge(gitCmd git.Git, repoURL, prRepoURL, branch, commit, branchDest
 	return nil
 }
 
+func listBranches(gitCmd git.Git) ([]string, error) {
+	if err := run(gitCmd.Fetch()); err != nil {
+		return nil, err
+	}
+	branches, err := gitCmd.Branch("-r").RunAndReturnTrimmedCombinedOutput()
+	if err != nil {
+		return nil, err
+	}
+
+	return strings.Split(branches, " "), nil
+}
+
+// Contains tells whether slice contains val.
+func Contains(slice []string, val string) bool {
```

Comment:
There is  a helper function for this in sliceutil pacakge: https://github.com/bitrise-io/go-utils/blob/6cc2aa4d248adef509996ac484bffd38856ddf80/sliceutil/sliceutil.go#L32